packages/truffle/test/scenarios/commands/unbox.js

```diff
@@ -153,6 +153,33 @@ describe("truffle unbox [ @standalone ]", () => {
           );
         }).timeout(20000);
       });
+
+      describe("recipes", () => {
+        it("modifies box according to specified recipe", async () => {
+          const cmd =
+            "unbox realcliffzhang/truffle-box-example --recipe React,TypeScript";
```

Comment:
Will create a new box repo under @truffle-box after changes are approved, and swap my own out for the new one.